src/Hal/Component/Cache/Cache.php

```diff
@@ -1,42 +0,0 @@
-<?php
```

Comment:
it was no longer used anywhere in the code base (i checked carefully). can easily put it back if you think it needed? I thought it is not a BC break because phpmetrics is an independent "app", not like a library with a public interface that needs a major version bump for a change like that... maybe i was wrong?